valohai/prepare_impl.py

```diff
@@ -24,12 +25,14 @@ def prepare(
     :param default_inputs: Dict of inputs with (list of) default URIs
     :param image: Default docker image
     :param environment: Default environment ID or slug
+    :param multifile: allow step to be prepared from multiple different source files
```

Comment:
would `multisource` be better to tie it with `source-path`? :thinking: 

just throwing it out there, I think the original should even be called something like `entry-file`, `entrypoint` or something but **it is what it is** :laughing:

Reply:
it would be a bit better to make the yaml field and the prepare argument names more aligned, but I think this is also good enough...